yt_dlp/extractor/tumblr.py

```diff
@@ -86,18 +82,28 @@ class TumblrIE(InfoExtractor):
             'like_count': int,
             'comment_count': int,
             'repost_count': int,
+            'thumbnail': r're:^https?://.*\.jpg',
+            'timestamp': 1455940159,
+            'view_count': int,
         },
         'add_ie': ['Vine'],
     }, {
-        'url': 'http://vitasidorkina.tumblr.com/post/134652425014/joskriver-victoriassecret-invisibility-or',
-        'md5': '01c12ceb82cbf6b2fe0703aa56b3ad72',
+        'url': 'https://silami.tumblr.com/post/84250043974/my-bad-river-flows-in-you-impression-on-maschine',
+        'md5': '3c92d7c3d867f14ccbeefa2119022277',
         'info_dict': {
-            'id': '-7LnUPGlSo',
+            'id': 'nYtvtTPuTl',
             'ext': 'mp4',
-            'title': 'Video by victoriassecret',
-            'description': 'Invisibility or flight…which superpower would YOU choose? #VSFashionShow #ThisOrThat',
-            'uploader_id': 'victoriassecret',
-            'thumbnail': r're:^https?://.*\.jpg'
+            'title': 'Video by silbulterman',
+            'description': '#maschine',
+            'uploader_id': '242859024',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'timestamp': 1398801174,
+            'like_count': int,
+            'uploader': 'Sil',
+            'channel': 'silbulterman',
+            'comment_count': int,
+            'upload_date': '20140429',
+            'duration': 0.0,  # [sic] (per instagram itself, not just the extractor)
```

Comment:
return `duration or None`

Reply:
In the Instagram extractor? Can do, but it will take some time to update and run the Instagram tests due to the rate-limiting. Shall I make a separate PR for it?

Reply:
```suggestion
```
Correct, but broken test is better than incorrect, passing test